style.css

```diff
@@ -1,6 +1,6 @@
 @import url('https://rsms.me/inter/inter.css');
 
-@media (max-width: 500px) {
```

Comment:
I think it would be better to use max-width:600px than 500px 🤗

Reply:
Good job here, your comment is polite and the use of the emoji makes it more friendly. Next time don't forget to add the reason why you are requesting the change 👍